hbs-helpers.js

```diff
@@ -42,12 +42,14 @@ function localizedBreachDataClasses(supportedLocales, dataClasses, args) {
 
 
 function fluentNestedBold(supportedLocales, id, args) {
-  for (const key in args.hash) {
-    if (["breachName", "breachCount"].includes(key)) {
-      args.hash[key] = `<span class="medium">${args.hash[key]}</span>`;
-    }
+  if (args.hash.breachName) {
+    args.hash.breachName = `<span class="medium">${args.hash.breachName}</span>`;
   }
-  return LocaleUtils.fluentFormat(supportedLocales, id, args.hash);
+  let localizedStrings = LocaleUtils.fluentFormat(supportedLocales, id, args.hash);
+  if (args.hash.breachCount) {
+    localizedStrings = localizedStrings.replace(/\d+/, `<span class="medium">${args.hash.breachCount}</span>`);
```

Comment:
My gut is telling me this may not work for small edge cases like "zero" (ref https://github.com/mozilla/blurts-server/issues/786#issuecomment-465789528).

But maybe that isn't relevant if the "zero" isn't in the arg hash. I guess this would only potentially impact these (which were the only instances of "zero" that I found in the app.ftl files):

https://github.com/mozilla/blurts-server/blob/c24b817c8510dac8fe6b4a1445459a9ce98f9bdd/locales/en/app.ftl#L298-L299

Reply:
Actually, another thing that occurred to me was if we're assuming any digit is a `$breachCount`, do we need to worry about any potential email addresses or breach names with numbers in them?

I found these 2 instances of `$breachCount` w/ `$featuredBreach`:

https://github.com/mozilla/blurts-server/blob/1388d1406613059921718503f71ee2b8c43bca48/locales/en/app.ftl#L149-L161

And this 1 instance of `$breachCount` w/ `$userEmail`:

https://github.com/mozilla/blurts-server/blob/1388d1406613059921718503f71ee2b8c43bca48/locales/en/app.ftl#L301-L305

And a few breach names that happen to have numbers in the name/title:

- [Collection #1](https://fx-breach-alerts.herokuapp.com/?breach=collection1)
- [Hub4Tech](https://fx-breach-alerts.herokuapp.com/?breach=hub4tech)
- [17173](https://fx-breach-alerts.herokuapp.com/?breach=17173)
- [2fast4u](https://fx-breach-alerts.herokuapp.com/?breach=2FAST4U)
- [000webhost](https://fx-breach-alerts.herokuapp.com/?breach=000WEBHOST)
- [7k7k](https://fx-breach-alerts.herokuapp.com/?breach=7K7K)
- [B2B USA BUSINESSES](https://fx-breach-alerts.herokuapp.com/?breach=B2BUSABUSINESSES)
- [Bell 2017 breach](https://fx-breach-alerts.herokuapp.com/?breach=BELL2017)
- [R2 2017 breach](https://fx-breach-alerts.herokuapp.com/?breach=r2-2017)